### ISSUE TITLE
ServicePointManager.ClientCipherSuitesCallback is obsolete in Mono

### ISSUE THREAD

**Issue by dwhathaway** [paulcbetts/ModernHttpClient#216]
ServicePointManager.ClientCipherSuitesCallback has been marked as Obsolete in Mono, and Android builds of ModernHttpClient no longer compile.

https://github.com/mono/mono/blob/86b5cde6c07809d81b592973f002ecf16cc22bf6/mcs/class/System/System.Net/ServicePointManager.extra.cs#L22-L23

This change is already in the current stable release of Xamarin.Android (6.1.0.71)

**Comment by fela98** [comment]
+1

**Comment by samofwise** [comment]
+1

**Comment by jimhanmer** [comment]
+1

**Comment by Cheesebaron** [comment]
If you don't have anything constructive to say, then don't do it. +1 isn't constructive and you should add a reaction to the opening post instead.

**Comment by jimhanmer** [comment]
Thanks for the advice - an error of inexperience. +1 removed now and reaction added.

**Comment by tharanagaraj** [comment]
@dwhathaway were you able to get it to compile?

**Comment by dwhathaway** [comment]
@tharanagaraj No, I haven't.  I haven't had a need to recompile from source since I initially ran into this.  The current available compiled versions do continue to work (at least they don't crash), despite the deprecation of this API.

**Comment by 2012MM** [comment]
How do we compile the update to support TLS 1.1+ on older version of Android if ServicePointManager is obsolete? Does anyone have an updated fork or branch supporting those changes in Issue [#173](https://github.com/paulcbetts/ModernHttpClient/issues/173)

**Comment by flchaux** [comment]
Any news on it ?
